include/envoy/server/admin.h

```diff
@@ -11,6 +11,9 @@
 namespace Envoy {
 namespace Server {
 
+#define MAKE_HANDLER(X)                                                                            \
+  [this](const std::string& url, Buffer::Instance& data) -> Http::Code { return X(url, data); }
```

Comment:
I have to admit I am not fully aware about how to make that call. Open to learn how you think about it :)